src/os/bluestore/bluefs_types.cc

```diff
@@ -81,20 +81,21 @@ bluefs_super_t::bluefs_super_t() : version(0), block_size(4096) {
 
 void bluefs_super_t::encode(bufferlist& bl) const
 {
-  ENCODE_START(3, 1, bl);
+  ENCODE_START(4, 1, bl);
   encode(uuid, bl);
   encode(osd_uuid, bl);
   encode(version, bl);
   encode(block_size, bl);
   encode(log_fnode, bl);
   encode(memorized_layout, bl);
   encode(bluefs_max_alloc_size, bl);
+  encode(reserved, bl);
```

Comment:
Some cleanup is needed. We add `reserved` (ver 4) to remove it alongsize `bluefs_max_alloc_size` in later commits (ver 4->ver 2)

Reply:
Ver 4 introduction has been removed. 
Ver 3 rolled back to v2 as bluefs_max_alloc_size has been removed as well and we don't need to provide backward compatibility for it so far.